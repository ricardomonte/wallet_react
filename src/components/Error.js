import React from 'react';
import { Headline3} from '@material/react-typography';
import '@material/react-typography/dist/typography.css';

const Errors = () => (
  <div className='loading_errors'>
    <Headline3>We are sorry, something went wrong</Headline3>
    <Headline3>We are working to fix it</Headline3>
  </div>
);

export default Errors;