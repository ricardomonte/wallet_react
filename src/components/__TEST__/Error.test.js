import React from 'react';
import '@testing-library/jest-dom';
import { render,screen } from '@testing-library/react';
import Error from '../Error';

test('Content from BtcAmount matches snapshot', () => {
  const content = render(<Error />);
  expect(content.container).toMatchSnapshot();
  expect(screen.getByText('We are sorry, something went wrong')).toBeInTheDocument();
});