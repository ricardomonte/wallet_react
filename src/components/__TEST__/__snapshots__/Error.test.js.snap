// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content from BtcAmount matches snapshot 1`] = `
<div>
  <div
    class="loading_errors"
  >
    <h3
      class="mdc-typography mdc-typography--headline3"
    >
      We are sorry, something went wrong
    </h3>
    <h3
      class="mdc-typography mdc-typography--headline3"
    >
      We are working to fix it
    </h3>
  </div>
</div>
`;
